spec/fixtures/pam_common_noninteractive_session.defaults.debian7

```diff
@@ -0,0 +1,6 @@
+# This file is being maintained by Puppet.
+# DO NOT EDIT
+session [default=1]   pam_permit.so
```

Comment:
Is all of the PAM info from a default installation? Our goal is that when you include the pam class on a freshly built system, it should not change anything. That way it is safe to use as-is without worrying if you are running a non-stock and possibly unsupported configuration.